projects/demo/src/modules/experimental/pin/pin.template.html

```diff
@@ -0,0 +1,83 @@
+<tui-doc-page
+    header="Pin"
+    package="EXPERIMENTAL"
+    type="components"
+>
+    <ng-template pageTab>
+        <tui-notification status="warning">
+            This code is
+            <strong>experimental</strong>
+            and is a subject to change. Expect final solution to be shipped in the next major version
+        </tui-notification>
+
+        <p class="tui-space_vertial-4">
+            Pins are used to show a location on a 2D plane. Use
+            <code>color</code>
+            ,
+            <code>background</code>
+            ,
+            <code>border</code>
+            and
+            <code>box-shadow</code>
+            to customize the pin
+        </p>
+
+        <tui-notification>
+            Pins are designed to be absolutely positioned on map or similar medium, therefore specifically their center
+            is placed where you put them.
+        </tui-notification>
+
+        <tui-doc-example
+            id="default"
+            heading="Default"
+            [content]="example1"
+            [fullsize]="true"
+        >
+            <tui-pin-example-1></tui-pin-example-1>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="dot"
+            heading="Dot"
+            [content]="example2"
+            [fullsize]="true"
+        >
+            <tui-pin-example-2></tui-pin-example-2>
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="selected"
+            heading="Selected"
```

Comment:
Input property (which is described in this example) is called `open`. Should we rename `id` and `heading` ? 🤔